docs/topics/forms/formsets.txt

```diff
@@ -748,6 +748,9 @@ argument - the index of the form in the formset. The index is ``None`` for the
     ...         kwargs['custom_kwarg'] = index
     ...         return kwargs
 
+    >>> ArticleFormSet = formset_factory(MyArticleForm, formset=BaseArticleFormSet)
+    >>> formset = ArticleFormSet()
```

Comment:
@xrmx Thanks for this patch :+1: However, I don't think this example is missing anything. It's about providing a custom `get_form_kwargs()` method and instantiating a formset is not important here. I'd rather remove an unnecessary import, i.e. `from django.forms import formset_factory`.

Reply:
I have proposed the patch because I was poking with formsets for the first time and I'd very much appreciated having the bits that showed me how to use the new formset available contextually. AFAIR most (all?) of the other examples on this page are self contained and don't assume prior knowledge on how to connect the dots. But hey it's not me who is maintaing this stuff so I'll update it if you ask so.

Reply:
> AFAIR most (all?) of the other examples on this page are self contained and don't assume prior knowledge on how to connect the dots.

IMO, this one is also self-contained. We can wait for the third opinion. Thanks!